### PR TITLE
[TC] Skip docker bootstrap TCs due to bz #2184673

### PR DESCRIPTION
**Pull request by vamahaja**

### CODE REVIEW

**Review by tintumathew10 — COMMENTED** [review]
LGTM

**Comment by vamahaja** [comment]
Testruns -

    http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-yam5z
    http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-8v5vg

File: tests/cephadm/test_remove_cluster.py [hunk 1]
```diff
@@ -105,6 +107,11 @@ def run(ceph_cluster, **kw):
                 f"Failed to execute rm-cluster on node '{node.ip_address}'"
             )
 
+    # Check if validation set to `False`
+    if not validation:
```
Comment (pranavprakash20) [inline tests/cephadm/test_remove_cluster.py:111]: please check if the validation variable is string or bool.

**Comment by vamahaja** [comment]
Tesrun - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-lyd58/

**Review by MohitBis — APPROVED** [review]
LGTM

**Pull request merged**